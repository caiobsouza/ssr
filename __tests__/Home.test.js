/* eslint-env jest */

import React from 'react';
import renderer from 'react-test-renderer';

import Home from '../src/components/Home';

describe('With Snapshot Testing', () => {
    it('App shows "Hello world!"', () => {
        const component = renderer.create(<Home />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
