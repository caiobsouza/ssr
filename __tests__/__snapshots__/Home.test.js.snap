// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello world!" 1`] = `
<h2>
  This is the home page!
</h2>
`;
